provider/labels.go

```diff
@@ -36,40 +44,53 @@ func fixLabelNames(prov *tfbridge.ProviderInfo) {
 				info.Elem.Fields = make(map[string]*tfbridge.SchemaInfo, obj.Len())
 			}
 			obj.Range(func(key string, m shim.Schema) bool {
-				i, ok := info.Fields[key]
+				i, ok := info.Elem.Fields[key]
 				if !ok {
 					i = new(tfbridge.SchemaInfo)
 					info.Elem.Fields[key] = i
 				}
-				apply(key, m, i)
+				currentPath = append(currentPath, key)
```

Comment:
We forgot a key translation.
```suggestion
				currentPath = append(currentPath,
					tfbridge.TerraformToPulumiNameV2(key, obj, info.Elem.Fields))

```